lisa/target.py

```diff
@@ -664,6 +665,68 @@ def install_tools(self, tools):
             self.target.install(tool)
             self._installed_tools.add(tool)
 
+    def build_kmodule(self, kmodule):
+        """
+        Compile kernel module that we want to load at the target at some point
+
+        :param kmodule: The name of the module to build
+        :type kmodule: str
+        """
+        logger = self.get_logger()
```

Comment:
`get_logger()` has to exist as a classmethod so it can be used from classmethods, but we could have a property on top when called in regular methods if we want.